key-export-common/src/lib.rs

```diff
@@ -0,0 +1,198 @@
+//! Dfns Key Export SDK
+//! Provides a basic functionality for key export.
+
+#![no_std]
+#![forbid(missing_docs)]
+
+extern crate alloc;
+
+use alloc::{string::String, vec::Vec};
+use serde_with::{base64::Base64, serde_as};
+
+use generic_ec::{curves::Secp256k1, Curve, NonZero, Point, Scalar, SecretScalar};
+
+/// Format of a decrypted key share
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[serde(bound = "")]
+pub struct KeySharePlaintext<E: Curve> {
+    /// The index (evaluation point)
+    pub index: NonZero<Scalar<E>>,
+    /// The secret share
+    pub secret_share: SecretScalar<E>,
+}
+
+/// Format of a decrypted key share
+#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+pub struct KeySharePlaintextNoGen {
+    /// The index (evaluation point)
+    // pub index: NonZero<Scalar<Secp256k1>>,
+    // /// The secret share
+    pub secret_share: SecretScalar<Secp256k1>,
+}
+
+/// Identity and encrypted share of a signer.
+#[serde_as]
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+pub struct EncryptedShareAndIdentity {
+    /// Signer's identity
+    #[serde_as(as = "Base64")]
+    pub signer_identity: Vec<u8>,
+    /// Signers's key share.
+    /// It is an encrypted `dfns_key_export_common::KeySharePlaintext`.
+    /// Ciphertext and plaintext are in format defined by `dfns-key-import-common`
+    /// library. See https://github.com/dfns-labs/trusted-dealer/
```

Comment:
You need to put a new line to separate a header from the body of the comment. Also the link can be prettified

```suggestion
    /// Signers's key share.
    ///
    /// It is an encrypted `dfns_key_export_common::KeySharePlaintext`.
    /// Ciphertext and plaintext are in format defined by `dfns-key-import-common`
    /// library. See [here](https://github.com/dfns-labs/trusted-dealer/).
```